sdk/maps/azure-maps-search/azure/maps/search/_api_version.py

```diff
@@ -1,29 +0,0 @@
-# ------------------------------------
```

Comment:
Why do we remove api_version? Can you justify?

Reply:
We had `_version.py` file, hence removed the `_api_version.py`